ext/node/ops/sqlite/database.rs

```diff
@@ -427,6 +427,14 @@ impl DatabaseSync {
     })
   }
 
+  // Returns "node:sqlite" for Node.js compatibility
+  #[getter]
+  #[symbol("sqlite-type")]
```

Comment:
@littledivy this seems to panic now... Should we fix it in `deno_core` and for the time being use the previous version in JS?

Reply:
oh we reuse the name as a Rust identifier in the macro. I think we can revert to the JS version for now @himself65. I will open a deno_core PR to fix the panic.